src/Container/Tarball.hs

```diff
@@ -204,6 +204,11 @@ isSymLink :: Tar.Entry -> Bool
 isSymLink (TarEntry.Entry _ (SymbolicLink _) _ _ _ _) = True
 isSymLink _ = False
 
+-- | True if tar entry is for a symbolic link, otherwise False.
```

Comment:
Should this read "True if tar entry is for a hard link, otherwise False."?

Reply:
yes - lazy copy/paste error. fixed.